apps/main-landing/src/components/top-bar/components/section.tsx

```diff
@@ -18,7 +18,7 @@ export const Section = ({ onItemClick, className, items = [] }: Properties) => {
           <li key={index} className="group">
             <NavigationMenu.Link asChild>
               <span>
-                <Link href="/#" passHref legacyBehavior>
+                <Link href={`/${item.url}`} passHref legacyBehavior>
```

Comment:
we probably should prefix them with `/` inside constants as these sections are tied to homepage

Reply:
I just gave you a comment about removing `a` tag because it's internal navigation but here I did the same thing :D can you remove it?

Reply:
sure :D